clusterloader2/testing/nichotr/config.yaml

```diff
@@ -0,0 +1,143 @@
+#Constants
```

Comment:
Nit, typically best to add a space after your comments. File formatters in goland or vscode should do this for you.

Reply:
Sounds good.